### PR TITLE
Adapt Jaus Ros Bridge Test

### PR DESCRIPTION
**Pull request by patapufate**
# Description

This PR fix the Jaus Ros Bridge test:
- Adapt Jaus ros bridge test
- Change mission to be used with Jaus Ros Bridge

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] The code builds clean without any errors or warnings
- [x] Unit tests are passing
- [x] Linter tests are passing

# How To Test

```sh
rostest mission_control test_jaus_ros_bridge_interface_mission_control.test 
```

### CODE REVIEW

File: mission_control/test/test_jaus_ros_bridge_interface_mission_control.py [hunk 1]
```diff
@@ -121,59 +124,55 @@ def test_jaus_ros_bridge_interface(self):
...
-        mission_to_execute.mission_id = 1
-        self.simulated_mission_control_execute_mission_pub.publish(mission_to_execute)
-        def mission_status_is_reported():
-            return ReportExecuteMissionState.EXECUTING in self.report_execute_mission.execute_mission_state
+        self.mission.load_mission("mission.xml")
+        self.mission.execute_mission()
+
+        # Wait for the mission to report EXECUTING
+        def executing_mission_status_is_reported():
+            return (ReportExecuteMissionState.EXECUTING in self.mission.execute_mission_state)
```
Comment (hidmic) [inline mission_control/test/test_jaus_ros_bridge_interface_mission_control.py:141]: @patapufate nit: no need for parenthesis

File: mission_control/test/test_jaus_ros_bridge_interface_mission_control.py [hunk 2]
```diff
@@ -121,59 +124,55 @@ def test_jaus_ros_bridge_interface(self):
...
-            wait_for(mission_status_is_reported),
+            wait_for(executing_mission_status_is_reported),
             msg='Mission control must report EXECUTING')
 
         # Abort mission and wait for it to wrap up
         abortMission = AbortMission()
         abortMission.mission_id = 1
         self.simulated_abort_mission_msg_pub.publish(abortMission)
+
         def complete_mission_status_is_reported():
```
Comment (hidmic) [inline mission_control/test/test_jaus_ros_bridge_interface_mission_control.py:151]: @patapufate nit: rename !

File: mission_control/test/test_jaus_ros_bridge_interface_mission_control.py (hunk 1 above)
Comment (patapufate) [reply mission_control/test/test_jaus_ros_bridge_interface_mission_control.py:141]: Fixed

File: mission_control/test/test_jaus_ros_bridge_interface_mission_control.py (hunk 2 above)
Comment (patapufate) [reply mission_control/test/test_jaus_ros_bridge_interface_mission_control.py:151]: Fixed!

File: mission_control/test/test_jaus_ros_bridge_interface_mission_control.py [hunk 3]
```diff
@@ -121,59 +124,55 @@ def test_jaus_ros_bridge_interface(self):
         self.mission_to_load.mission_file_full_path = "NO_MISSION"
         self.simulated_mission_control_load_mission_pub.publish(
             self.mission_to_load)
```
[the remark this answers is not in the document]
Comment (patapufate) [reply mission_control/test/test_jaus_ros_bridge_interface_mission_control.py:126]: Fixed!

File: mission_control/test/mission_interface.py [hunk 4]
```diff
@@ -108,10 +108,11 @@ def load_mission(self, mission_name):
         self.mission_load_state_flag = False
         mission_to_load = LoadMission()
         mission_to_load.mission_file_full_path = self.full_path
-        self.simulated_mission_control_load_mission_pub.publish(mission_to_load)
+        self.simulated_mission_control_load_mission_pub.publish(
+            mission_to_load)
 
         def load_mission_state():
-            return self.mission_load_state == ReportLoadMissionState.SUCCESS
+            return self.mission_load_state == load_status
```
Comment (hidmic) [inline mission_control/test/mission_interface.py:115]: @patapufate nit^N: if `MissionInterface.load_mission()` would wait and return the load state, the test would read better e.g. `self.assertEq(load_mission(...), SUCCESS)`

File: mission_control/test/test_jaus_ros_bridge_interface_mission_control.py [hunk 5]
```diff
@@ -148,10 +139,10 @@ def executing_mission_status_is_reported():
         abortMission.mission_id = 1
         self.simulated_abort_mission_msg_pub.publish(abortMission)
 
-        def complete_mission_status_is_reported():
+        def aborting_mission_status_is_reported():
             return (ReportExecuteMissionState.ABORTING in self.mission.execute_mission_state)
```
Comment (hidmic) [inline mission_control/test/test_jaus_ros_bridge_interface_mission_control.py:143]: @patapufate nit: redundant parenthesis here too.

**Review by hidmic — APPROVED** [review]

File: mission_control/test/test_jaus_ros_bridge_interface_mission_control.py (hunk 5 above)
Comment (patapufate) [reply mission_control/test/test_jaus_ros_bridge_interface_mission_control.py:143]: Fixed

File: mission_control/test/mission_interface.py (hunk 4 above)
Comment (patapufate) [reply mission_control/test/mission_interface.py]: Changed

**Pull request merged**